### DISCUSSION TITLE
how to limit RAM of board?

### DISCUSSION THREAD

**Discussion by MattiaGrigoli** [Joshua-Riek/ubuntu-rockchip#1134] [Q&A]
is there a way to limit the RAM of the board? (ex: 16GB ram board limit to 4GB)
in the source code, kernel, or even a command with the image already built.
it could seem senseless but it's for the RGA issue

**Comment by nyanmisaka** [comment]
It's not related to the kernel. Modify in **u-boot** memory map.
